src/pipe_operator.rs

```diff
@@ -1,6 +1,6 @@
 use crate::util;
 
-pub fn pipe(command: String) {
+pub fn pipe(mut history:Vec<String>, command: String) {
```

Comment:
No need of mut

Reply:
updated the input parameter and removed assignment from the dispatch helper function to history.